### ISSUE TITLE
Recorder Extension only generate error.png

### ISSUE THREAD

**Issue by cabiste007** [Codeception/Codeception#5128]
#### What are you trying to achieve?
I want to make screenshots of every action I make and getting th index file
#### What do you get instead?
Every thing work smoothly and the test is passed but I get the steps screenshot with the last one named error.png without the index file 

all this happen when I add 

> $I->wantTo('move to Form1 ');

here is what console show
![image](https://user-images.githubusercontent.com/18129013/44334773-c00b6780-a472-11e8-93ea-9db28fd97d57.png)

```
> Provide test source code if related
here is my test code

### Details
and I get as output this 
![image](https://user-images.githubusercontent.com/18129013/44335069-9b63bf80-a473-11e8-9b82-5038282fe5a4.png)

instead of this 
![image](https://user-images.githubusercontent.com/18129013/44336037-6dcc4580-a476-11e8-81f2-21a05bc444ef.png)






here is acceptance.suite.yml
![image](https://user-images.githubusercontent.com/18129013/44334870-05c83000-a473-11e8-9b5d-e67dbbc3e813.png)

here is codeception.yml

![image](https://user-images.githubusercontent.com/18129013/44334947-36a86500-a473-11e8-9e54-a1ce62b2ffe7.png)

and thanks in advance


```

**Comment by Naktibalda** [comment]
It looks like your test fails before making any actions, so you have to solve that issue instead.

**Comment by Naktibalda** [comment]
Does it work correctly if you downgrade Codeception to 2.4.4?
If it does, then your issue was caused by #5101

**Comment by cabiste007** [comment]
ohh thanks it work dude ;)

**Comment by Naktibalda** [comment]
Ah, okay, so it will be fixed in 2.4.6.

@cabiste007 Please set version of Codeception to `2.4-dev`, run `composer update` and see if it works.

**Comment by cabiste007** [comment]
thanks guys
